gift-certificate-data/src/main/java/com/epam/esm/giftcertificatemodule3/entity/Order.java

```diff
@@ -0,0 +1,126 @@
+package com.epam.esm.giftcertificatemodule3.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Table(name = "orders")
+public class Order extends BaseEntity implements Serializable {
+
+    private static final long serialVersionUID = 1324820937632935342L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "order_id")
+    private Long id;
+
+    @Column(name = "date")
+    private OffsetDateTime date;
+
+    @Column(name = "price")
+    private Double price;
```

Comment:
Floating point numbers are not suitable for handling money. Please investigate money types in SQL and BigDecimal types in Java.